### ISSUE TITLE
False positive for convert function to ES6 class

### ISSUE THREAD

**Issue by andy-ms** [Microsoft/TypeScript#22240]
**TypeScript Version:**  2.8.0-dev.20180228

**Code**

```ts
function f() {}
f.prototype.bar = function(){
    this.x = 1;
};
```

Highlight the second `function` keyword. You will get a suggestion to convert *it* to an ES6 class -- actually attempting to apply the suggestion does not work.
This happens because apparently the function expression has a non-empty `.members` containing `x`.
